release-notes/april-2025.md

```diff
@@ -0,0 +1,104 @@
+---
+slug: april-2025
+title: April 2025
+hide_title: true
+description: >-
+    Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+tags:
+  - Release notes
+hide_table_of_contents: false
+date: 2025-04-30T10:00
+---
+
+# Semgrep release notes for April 2025
+
+<!-- Remember to update latest endpoint -->
+<!-- Remember to update index page -->
+
+The following updates were made to Semgrep in April 2025.
+
+## 🌐 Semgrep AppSec Platform
+
+### Added
+
+### Changed
+
+### Fixed
+
+## 💻 Semgrep Code
+
+### Added
+
+### Changed
+
+### Fixed
+
+## ⛓️ Semgrep Supply Chain
+
+### Added
+
+- [SBOM exports through the Semgrep API](https://semgrep.dev/api/v1/docs/#tag/SupplyChainService/operation/semgrep_app.products.sca.handlers.sbom.openapi_create_sbom_export) is now generally available.
+- Malicious dependency detection is now in **public beta**. Semgrep can detect malicious dependencies in your projects and in pull requests (PRs) or merge requests (MRs), allowing you to investigate its impact, identify indicators of compromise or security breach, and mitigate the threats.
+- **Upgrade guidance** and **click to fix** are now in **private beta** for users with Python projects hosted by GitHub.com and Semgrep Assistant enabled. With upgrade guidance and click to fix, Supply Chain analyzes your project to surface breaking changes that you must fix as part of a version upgrade. Semgrep AppSec Platform provides you with a one-click option that opens a pull request to:
+    1. Bump the version.
+    2. Lets the developer know if the upgrade is safe or if there are breaking changes and what those changes are.
+- **Transitive reachability** is now in **private beta**. For JavaScript projects, Semgrep reachability now extends to transitive dependencies.
+
+### Changed
+
+- Increased the rate limit for SBOM exports through the Semgrep API.
+
+## 🤖 Semgrep Assistant
+
+### Added
+
+- Semgrep Assistant now attempts to create a memory during triage if possible.
```

Comment:
```suggestion
- Semgrep Assistant now attempts to create a memory during triage if possible.
```
Is the user prompted with an Ok/Cancel (or similar) dialog?

Reply:
I'll add more info: the user gets a toast that says this happened, along with a link to memories